README.md

```diff
@@ -22,15 +22,16 @@ which is already locked, it will wait for the resource to be free.
 
 Each lockable resource has the following properties:
 
-- **Name** - A name (not containing spaces!) for this particular resource, i.e.
+- **Name** - A mandatory name (not containing spaces!) for this particular resource, i.e.
   `DK_Printer_ColorA3_2342`
-- **Description** - A verbose description of this particular resource,
-  i.e. ` Printers in the Danish Office`
-- **Labels** - Space-delimited list of Labels (Not containing spaces) used to
-  identify a pool of resources. i.e. `DK_Printers_Office`,
+- **Description** - A optional verbose description of this particular resource,
```

Comment:
Match the form used two lines later, avoid the mismatch between the article "A" (should be "An")
```suggestion
- **Description** - Optional verbose description of this particular resource,
```

Reply:
Thx for suggestion. I will do it also for other properties (to be consistent)